docs/source/usage.rst

```diff
@@ -6,9 +6,9 @@ Overview
 
```

Comment:
todo -- the module/method link to create_db in this page isn't working. address this